specification/schemas/ShipmentColliRelationship.json

```diff
@@ -8,7 +8,7 @@
     "data": {
       "type": "array",
       "items": {
-        "$ref": "#/components/schemas/Shipment"
+        "$ref": "#/components/schemas/ShipmentResource"
```

Comment:
I would expect a filename change in here as well then. Also, I'm not really sure about the 'resource' suffix since no other components use this.

Please elaborate if I'm missing something (:

Reply:
Good remark, and yes there is a difference between some resources in our api-specification.

Half of the resources have been untouched since we moved from Swagger 2.0 to 3.0 (OpenAPI). This update introduced `oneOf` and `allOf` which allowed us to combine partial resources. Then we came up with a more re-usable way of working: separate attribute requirements from resources. For example:
```json
{
  "allOf": [
    { "$ref": "#/components/schemas/Shipment" },
    { "required": ["recipient_address", "sender_address"] }
  ]
}
```
This new approach is explained in the PR that introduced it for three resources: https://github.com/MyParcelCOM/api-specification/pull/503
So far, we have reworked these resources: `Report` + `Shipment` + `Shop` + `Status` + `SystemMessage` + `User`.